scico/linop/radon_svmbir.py

```diff
@@ -190,20 +192,28 @@ class SVMBIRWeightedSquaredL2Loss(WeightedSquaredL2Loss):
     of :class:`scico.linop.Diagonal`. If :math:`W` is None, it is set to
     :class:`scico.linop.Identity`.
 
+    The extended loss differs from a typical weighted squared :math:`\ell_2` loss
+    is the following ways.
+
     When ``positivity=True``, the prox projects onto the non-negative
-    quadrant, but the the loss, :math:`\alpha l(\mb{y}, A(\mb{x}))`,
+    orthant, but the the loss, :math:`\alpha l(\mb{y}, A(\mb{x}))`,
     is unaffected by this setting and still evaluates to finite values
-    when :math:`\mb{x}` is not in the non-negative quadrant.
+    when :math:`\mb{x}` is not in the non-negative orthant.
+
+    When the ``is_masked`` option of the associated :class:`.ParallelBeamProjector` is True,
+    the reconstruction is computed over a masked region of the image as described
+    in class :class:`.ParallelBeamProjector`.
     """
 
     def __init__(
         self,
         *args,
-        prox_kwargs: dict = {"maxiter": 1000, "ctol": 0.001},
+        prox_kwargs: dict = {},
```

Comment:
initializing with a mutable type is risky, I suggest `=None` then `if prox_kwargs is None:`.

Reply:
Check out `admm.py`

```python
    def __init__(self, cg_kwargs: Optional[dict] = None, cg_function: str = "scico"):
        """Initialize a :class:`LinearSubproblemSolver` object.

        Args:
            cg_kwargs: Dictionary of arguments for CG solver. See
                documentation for :func:`scico.solver.cg` or
                :func:`jax.scipy.sparse.linalg.cg`,
                including how to specify a preconditioner.
                Default values are the same as those of
                :func:`scico.solver.cg`, except for
                ``"tol": 1e-4`` and ``"maxiter": 100``.
            cg_function: String indicating which CG implementation to
                use. One of "jax" or "scico"; default "scico".  If
                "scico", uses :func:`scico.solver.cg`. If "jax", uses
                :func:`jax.scipy.sparse.linalg.cg`. The "jax" option is
                slower on small-scale problems or problems involving
                external functions, but can be differentiated through.
                The "scico" option is faster on small-scale problems, but
                slower on large-scale problems where the forward
                operator is written entirely in jax.
        """

        default_cg_kwargs = {"tol": 1e-4, "maxiter": 100}
        if cg_kwargs:
            default_cg_kwargs.update(cg_kwargs)
        self.cg_kwargs = default_cg_kwargs
```

Reply:
Fixed it.